src/www/ui_components/server_card/index.ts

```diff
@@ -17,6 +17,18 @@ import {LegacyElementMixin} from "@polymer/polymer/lib/legacy/legacy-element-mix
 
 import {ServerConnectionState} from "./server_connection_viz";
 
+export * from "./server_connection_viz";
+
+export interface ServerCardModel {
```

Comment:
Since we were talking about slots, maybe i move this back to the server_list and call it "ServerListItemModel"